app/controllers/content_items_controller.rb

```diff
@@ -26,8 +26,6 @@ def present(content_item)
     presenter_name = content_item['schema_name'].classify + 'Presenter'
     presenter_class = Object.const_get(presenter_name)
     presenter_class.new(content_item)
-  rescue NameError
```

Comment:
because it does nothing and masks real errors, everytime you have an error anywhere in your presenter, it gives the reraised error, which is not helpful